Doc/c-api/init.rst

```diff
@@ -622,7 +622,8 @@ Process-wide parameters
       It now returns ``NULL`` if called before :c:func:`Py_Initialize`.
 
    .. deprecated-removed:: 3.13 3.15
```

Comment:
I think that we should postpone hard deprecation until there is a non-deprecated replacement that works in all maintained Python versions.

Reply:
PySys_GetObject() works on all Python versions, it's not deprecated: https://docs.python.org/dev/c-api/sys.html#c.PySys_GetObject

Reply:
You can use pythoncapi-compat to get PyConfig_Get() on Python 3.13 and older.